src/Orleans/Properties/AssemblyInfo.cs

```diff
@@ -31,6 +31,7 @@
 [assembly: InternalsVisibleTo("OrleansHost")]
 [assembly: InternalsVisibleTo("OrleansAzureUtils")]
 [assembly: InternalsVisibleTo("OrleansAWSUtils")]
+[assembly: InternalsVisibleTo("OrleansGoogleUtils")]
```

Comment:
Ideally we should avoid using internal types in Orleans for extensions. Is there a way to avoid this?